Magento2/Sniffs/GraphQL/ValidTypeNameSniff.php

```diff
@@ -32,7 +32,7 @@ public function process(File $phpcsFile, $stackPtr)
         //compose entity name by making use of the next strings that we find until we hit a non-string token
         $name = '';
         for ($i=$stackPtr+1; $tokens[$i]['code'] === T_STRING; ++$i) {
-            $name .= $tokens[$i]['content'];
+            $name .= rtrim($tokens[$i]['content']);
```

Comment:
Do we want to also allow white-space on the left of the name?
```suggestion
            $name .= trim($tokens[$i]['content']);
```

Reply:
I think that tokenizer is never gonna create a T_STRING token with a whitespace on the left.

Reply:
We should either fix the tokeniser to never include (leading nor) trailing whitespace in a T_STRING token, or run this through `trim()`. Using `rtrim()` seems incomplete.

Reply:
Thanks for the explanation and highlighting where the extra white-space comes from (ie, the GraphQL tokeniser within this repository). Let's add a code comment here so others know why removing the trailing newline character(s) is safe in this context.